### PR TITLE
Change `sqlite3_busy_timeout` to 60 seconds, add warnings re: `litestream`.

### PR DESCRIPTION
**Pull request by ZmnSCPxj**
Fixes: #4860 

Fixes: #4857 if @x3nb63 can confirm

We need to specify versions in `BACKUP.md` since it is published on https://lightning.readthedocs.io/BACKUP.html and users might assume the recommendation applies to all versions; `litestream` will reliably work after 5bf2b5c0ba **and** on reasonably fast media/low-load conditions.

### CODE REVIEW

File: wallet/db_postgres.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include "db_postgres_sqlgen.c"
 #include <ccan/ccan/tal/str/str.h>
+#include <ccan/compiler/compiler.h>
```
Comment (vincenzopalazzo) [inline wallet/db_postgres.c:3]: unable to understand the motivation that we need the `ccan/compiler` here

Comment (ZmnSCPxj) [reply wallet/db_postgres.c:3]: `UNUSED` macro.

**Comment by rustyrussell** [comment]
Let's just increase the default to 30 or 60 seconds. I mean, we could do better, but the user surely prefers a delay to a crash!

Providing knobs like this is a last resort, IMHO.

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4867/commits/2434e4d67f14f9261203a5e16482da2102cc17e5

**Review by rustyrussell — APPROVED** [review]
Ack 2434e4d67f14f9261203a5e16482da2102cc17e5

**Pull request merged**